Sources/Purchasing/ReceiptFetcher.swift

```diff
@@ -53,33 +53,50 @@ class ReceiptFetcher {
 
 }
 
-private extension ReceiptFetcher {
+extension ReceiptFetcher {
 
-    func receiptData() -> Data? {
-        guard var receiptURL: URL = systemInfo.bundle.appStoreReceiptURL else {
+    var receiptURL: URL? {
+        guard var receiptURL = self.systemInfo.bundle.appStoreReceiptURL else {
             Logger.debug(Strings.receipt.no_sandbox_receipt_restore)
             return nil
         }
 
         #if os(watchOS)
+        receiptURL = self.watchOSReceiptURL(receiptURL)
+        #endif
+
+        return receiptURL
+    }
+
+    func watchOSReceiptURL(_ receiptURL: URL) -> URL? {
```

Comment:
[not for this PR] I wonder if it'd make sense to move this one to `StoreKitWorkarounds`

Reply:
Oh yeah! Let me do that

Reply:
Done